src/rich-text-editor.component.js

```diff
@@ -192,6 +184,26 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
 
     return html
   }
+
+  function setHTML(html) {
+    emptyEditor()
+    divRef.current.innerHTML = richTextContext.sanitizeHTML(html, 'setHTML')
```

Comment:
What is the purpose of `'setHTML'` here?

Reply:
Oh, good question.  The thought behind passing back a string to the sanitizer (along with the htmlString itself) was to give the sanitizer some context on what bandicoot was doing with html.  That way _if_ the sanitizer wanted to sanitize the html slightly differently based on what was happening, it could.  For example, maybe generally the sanitizer wanted to enforce some rule that the htmlString always started and ended with angle brackets, but when pasting text, it might not have angle brackets, and the if implementer was willing to make an exception in that case it could make conditional exceptions based on what bandicoot was doing.  Anyway, that was the thought at least.

Reply:
Missed this in the docs but a potentially useful add.

Reply:
But we don't provide this string anywhere else that we call sanitizeHTML.

I think we should leave this off until there's a use case that calls for it.

Reply:
I'm not sure what you mean by we don't provide this string anywhere else we call sanitizeHTML.  We provide a string for all of the places that call sanitizeHTML.  They are just all different: 'setHTML', 'getHTML', 'insertContentEditableFalseHTML', 'pasteHTML', 'setLastSavedHTML'.

Am I miss understanding your comment?

Also, Canopy might have a use for it.

Reply:
What use?

Reply:
What use?

I prefer a "has a use case" over a "might have a use case" 😄

Reply:
haha... well we haven't implemented it yet, but the thought was that on pasting we might allow text that isn't wrapped in < >... while when sanitizing other html we would enforce it does have to be wrapped in < >.  But we might not do it, so I can take it out of this merge request until we or someone else HAS to have it.

Reply:
I see - that makes sense. Let's keep it for now, and also add it to the documentation, including all the possible values that could be passed

Reply:
sounds good

Reply:
I'll go ahead and start drafting some stuff for the docs